lib/completion.dart

```diff
@@ -5,8 +5,8 @@ part 'completion.g.dart';
 @JsonSerializable(createFactory: false)
 class CompletionApiParameters {
   CompletionApiParameters(this.prompt,
-      {this.maxTokens = 16,
-      this.temperature = 1,
+      {this.maxTokens = 256,
```

Comment:
```suggestion
      {this.maxTokens = 16,
```
According to https://beta.openai.com/docs/api-reference/completions/create, 16 tokens and temperature of 1 is still the default parameter, which is why I would keep it like that.